src/board/m5stack/M5CORE2.h

```diff
@@ -0,0 +1,244 @@
+#define ESP_PANEL_TOUCH_MIRROR_X        (0)         // 0/1
+#define ESP_PANEL_TOUCH_MIRROR_Y        (0)         // 0/1
+
+/* Touch Other Settings */
+/* IO num of RESET pin, set to -1 if not use */
+#define ESP_PANEL_TOUCH_IO_RST          (-1)
+#define ESP_PANEL_TOUCH_RST_LEVEL       (0)         // 0: low level, 1: high level
+/* IO num of INT pin, set to -1 if not use */
+#define ESP_PANEL_TOUCH_IO_INT          (39)
+#define ESP_PANEL_TOUCH_INT_LEVEL       (0)         // 0: low level, 1: high level
+
+#endif /* ESP_PANEL_USE_TOUCH */
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////// Please update the following macros to configure the backlight ////////////////////////////
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+#define ESP_PANEL_USE_BACKLIGHT         (0)         // 0/1
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////// Please update the following macros to configure the IO expander //////////////////////////
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+/* Set to 0 if not using IO Expander */
+#define ESP_PANEL_USE_EXPANDER          (0)         // 0/1
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////// Please utilize the following macros to execute any additional code if required. //////////
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// #define ESP_PANEL_BEGIN_START_FUNCTION( panel )
+// #define ESP_PANEL_BEGIN_EXPANDER_START_FUNCTION( panel )
+// #define ESP_PANEL_BEGIN_EXPANDER_END_FUNCTION( panel )
+// #define ESP_PANEL_BEGIN_LCD_START_FUNCTION( panel )
+// #define ESP_PANEL_BEGIN_LCD_END_FUNCTION( panel )
+// #define ESP_PANEL_BEGIN_TOUCH_START_FUNCTION( panel )
+// #define ESP_PANEL_BEGIN_TOUCH_END_FUNCTION( panel )
+// #define ESP_PANEL_BEGIN_BACKLIGHT_START_FUNCTION( panel )
+// #define ESP_PANEL_BEGIN_BACKLIGHT_END_FUNCTION( panel )
+#define ESP_PANEL_BEGIN_END_FUNCTION( panel ) \
+    { \
+        static const char *TAG = "DEBUG"; \
+        static const uint8_t AXP_ADDR = 0x34; \
+        static const uint8_t I2C_MASTER_TIMEOUT_MS = 1000; \
+        static i2c_port_t i2c_master_port = I2C_NUM_0; \
+         \
+        uint8_t device_id = 0; \
+        uint8_t read_value = 0; \
+        uint8_t write_value = 0; \
+        uint8_t reg_addr = 0; \
+        uint8_t write_buf[2] = {0}; \
+         \
+        reg_addr = 0x03; \
+        i2c_master_write_read_device(i2c_master_port, AXP_ADDR, &reg_addr, 1, &device_id, 1, I2C_MASTER_TIMEOUT_MS / portTICK_PERIOD_MS); \
```

Comment:
```suggestion
        i2c_master_write_read_device(i2c_master_port, AXP_ADDR, (uint8_t []){ 0x03 }, 1, &device_id, 1, I2C_MASTER_TIMEOUT_MS / portTICK_PERIOD_MS); \
```
Here is a simple way without using `reg_addr`.

Reply:
Due to the language features of C++, it is not feasible to use temporary variables in macro functions and obtain their addresses, so I have maintained my original writing style.